ceph.spec.in

```diff
@@ -2121,11 +2121,9 @@ fi
 %{python3_sitelib}/ceph
 %{python3_sitelib}/ceph-*.egg-info
 
-%if 0%{with cephfs_shell}
```

Comment:
There's also a cephfs-shell package definition in this file that needs fixed too.

Reply:
Oh, ok...

Reply:
Fixed.